include/daycare.h

```diff
@@ -30,5 +30,9 @@ void SetDaycareCompatibilityString(void);
 bool8 NameHasGenderSymbol(const u8 *name, u8 genderRatio);
 void ShowDaycareLevelMenu(void);
 void ChooseSendDaycareMon(void);
+#if POKEDEX_PLUS_HGSS == TRUE
+u8 GetEggMovesSpecies(u16 species, u16 *eggMoves);
+bool8 SpeciesCanLearnEggMove(u16 species, u16 move);
+#endif
```

Comment:
These 2 functions might be useful to have available, regardless of the `POKEDEX_PLUS_HGSS` config, so I would remove the ifdefs here and in the definitions.